### ISSUE TITLE
Make a new release of simpcomp for GAP?

### ISSUE THREAD

**Issue by fingolfin** [simpcomp-team/simpcomp#55]
It would be nice if my open PRs could be merged and a new release be made with those and other changes made in the past 2.5 years.

Ping @simpcomp-team

**Comment by simpcomp-team** [comment]
Yes, absolutely. Thanks for pointing out all these outdated dependencies.

**Comment by fingolfin** [comment]
Thank you for merging my PRs. Unfortunately at least one had an issue, which was not visible because CI had been broken for some time. I've submitted another PR #58 which hopefully will fix this.

**Comment by fingolfin** [comment]
You might also like https://github.com/simpcomp-team/simpcomp/pull/59

**Comment by simpcomp-team** [comment]
Ah… I was hoping to deal with this after all the merges. Thanks for dealing with this already.
